webapp/shared/parking_site/parking_site_search_query.py

```diff
@@ -7,25 +7,28 @@
 from typing import Optional
 
 from parkapi_sources.models.enums import PurposeType
+from validataclass.dataclasses import Default
 from validataclass.exceptions import ValidationError
-from validataclass.validators import EnumValidator, IntegerValidator, ListValidator, NumericValidator, StringValidator
+from validataclass.validators import EnumValidator, ListValidator, NumericValidator, StringValidator
```

Comment:
Was IntegerValidator an unneeded import before? It has been removed now, but I don't see a removed usage(?)

Reply:
ATM it's commented out, see TODO below.